spec/features/commands_spec.rb

```diff
@@ -157,6 +157,18 @@
     end.to change { deployment_requests.count }.by(1)
   end
 
+  scenario 'checking a locked branch' do
+    # It says that it's locked after someone locks it.
+    command "/deploy lock staging on acme-inc/api: I'm testing some stuff", as: slack_accounts(:david)
+    command '/deploy check lock staging on acme-inc/api', as: slack_accounts(:david)
```

Comment:
that seems like it'd be useful. i updated this PR to change `check lock` to `check`. i'm not exactly sure how how to structure the attachments though: how would that look?